src/main/java/ch/njol/skript/expressions/ExprExperienceCooldownChangeReason.java

```diff
@@ -11,7 +11,7 @@
 
 @Name("Experience Cooldown Change Reason")
 @Description({
-	"The <a href='classes.html#experiencechangereason'>experience change reason</a> within in an" +
+	"The <a href='classes.html#experiencechangereason'>experience change reason</a> within in an " +
```

Comment:
```suggestion
	"The <a href='classes.html#experiencechangereason'>experience change reason</a> within an " +
```
how did you not notice this?

Reply:
Dang man, whoever added this Expr class in the first place sucks at English

Reply:
Ngl gotta agree with @TheAbsolutionism , also no idea how i didn't notice that, I was too focused on the missing space and on the masterpiece that the first attempt of this pr was.